drivers/wireless/ieee80211/bcm43xxx/Kconfig

```diff
@@ -40,6 +40,35 @@ config IEEE80211_INFINEON_CYW43439
 	default n
 	select IEEE80211_BROADCOM_HAVE_CLM
 
+config CYW43439_FIRMWARE_BIN_PATH
+	string "Path to Infineon 43439 firmware file"
+	depends on IEEE80211_INFINEON_CYW43439
+	default "${PICO_SDK_PATH}/lib/cyw43-driver/firmware/43439A0-7.95.49.00.combined"
+	---help---
+		This should be a path to a file containing both the cyw43439 firmware and
+		the CLB blob.   The firmware should be padded to a 256 byte boundary and
+		then the CLM blob should be appended.
+
+		If this file is updated, check the CYW43439_FIRMWARE_LEN and 
+		CYW43439_CLM_BLOB_LEN below to make sure they reflect the new, un-padded 
+		lengths of the two parts.
+
+config CYW43439_FIRMWARE_LEN
```

Comment:
why not pass len by gobal variable?

Reply:
The firmware file we get from pico-sdk actually contains two things concatenated together: the actual firmware and the clm blob.  These to parts are copied to the chip by different methods from different functions. Since the location of the firmware file is a configurable item, it seemed reasonable that the length of (at least) the first part was too.